data/saas/config/gorgias_config.yml

```diff
@@ -0,0 +1,134 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Gorgias SaaS Config
+  type: gorgias
+  description: A sample schema representing the Gorgias connector for Fides
+  version: 0.1.0
+
+  connector_params:
+    - name: domain
+      default_value: ethyca-test.gorgias.com
+    - name: api_key
+    - name: username
+
+  client_config:
+    protocol: https
+    host: <domain>
+    authentication:
+      strategy: basic
+      configuration:
+        username: <username>
+        password: <api_key>
+
+  test_request:
+    method: GET
+    path: /api/customers
+    query_params:
```

Comment:
We can't use the email identity for the test_request, it won't be available when we are first setting up the connection, you can use a dummy email or another endpoint that doesn't require any inputs.